### PR TITLE
🚢 App Router PoC - Products Pages

### LINKED ISSUE #2168: 🧪 Experiment - App Router w/ Preview Route
cc @wicksipedia 

In spite of previous failed attempts to migrate to the app router, I've thought of a way it can be done. As per https://github.com/SSWConsulting/SSW.Website/issues/2115#issuecomment-1940200744, the problem is that using `useTina` requires a client-side environment. This means the entire page would have to be client-side rendered to be able to use visual editing in `/admin`. 

A workaround to this is creating two separate routes:

* Server-side - `/page-route-here`
* Client-side  - `/preview/page-route-here` (would have to be hidden from SEO/public)

Then, TinaCMS can load the client-side page as a preview, and still have visual editing capabilities, without the server-rendered page being impacted by use of `useTina`. 

Previous attempt: https://github.com/SSWConsulting/SSW.Website/issues/1078

```[tasklist]
### Tasks
- [x] Create a POC for an app router page
```

### PR DESCRIPTION
**Pull request by Harry-Ross**
**TODO: Ensure that everything isn't just a client component**

* Migrated `/products/*` to App Router

- Affected routes: 

- Fixed #2168

- [ ] If adding a new page, I have followed the [📃 New Webpage](https://github.com/SSWConsulting/SSW.Website/issues/new?assignees=&labels=&projects=&template=new_webpage.yml&title=%F0%9F%93%84+%7B%7B+TITLE+%7D%7D+) issue template

- [ ] Include done video or screenshots

### CODE REVIEW

File: app/products/[filename]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import client from "@/tina/client";
+import { useSEO } from "hooks/useSeo";
+import { Metadata } from "next";
+import { draftMode } from "next/headers";
+import ProductsContent from "./products-content";
+import ProductsPreview from "./products-preview";
+
+export const dynamicParams = false; // False will not allow Next to generate any routes on request
```
Comment (wicksipedia) [inline app/products/[filename]/page.tsx]: add link to docs

File: app/layout.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    <html lang="en" className={openSans.className}>
+      <Head>
+        <meta charSet="utf-8" />
+        <meta name="viewport" content="initial-scale=1.0, width=device-width" />
+        <link rel="manifest" href="/site.webmanifest" />
+        <meta name="msapplication-TileColor" content="#cc4141" />
+        <meta name="theme-color" content="#ffffff" />
+      </Head>
+      <body>
+        <Analytics />
```
Comment (wicksipedia) [inline app/layout.tsx]: can we move this to the end of the body
also, it only uses google tag manager - for app router pages, can we use this instead - https://nextjs.org/docs/app/building-your-application/optimizing/third-party-libraries#google-tag-manager

**Review by wicksipedia — APPROVED** [review]
test pass with changes

File: app/products/[filename]/page.tsx (hunk 1 above)
Comment (amankumarrr) [reply app/products/[filename]/page.tsx]: done

**Pull request merged**